ompi/datatype/ompi_datatype.h

```diff
@@ -73,6 +73,7 @@ struct ompi_datatype_t {
 
     void*              args;                     /**< Data description for the user */
     void*              packed_description;       /**< Packed description of the datatype */
+    uint64_t           pml_data;                 /**< PML-specific information */
     /* --- cacheline 6 boundary (384 bytes) --- */
```

Comment:
Since you just added a uint64_t right above this, is this comment (cacheline boundary) still correct?